### PR TITLE
fix: [PIE-6091]: removing raw type from nexus3

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Review by ishantmehtaharness — APPROVED** [review]

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
trigger cypress

**Review by ishantmehtaharness — APPROVED** [review]

File: src/modules/70-pipeline/utils/stageHelpers.ts [hunk 1]
```diff
@@ -79,8 +79,8 @@ export enum RepositoryFormatTypes {
 export const nexus2RepositoryFormatTypes = [
   { label: 'Maven', value: RepositoryFormatTypes.Maven },
   { label: 'NPM', value: RepositoryFormatTypes.NPM },
-  { label: 'NuGet', value: RepositoryFormatTypes.NuGet },
-  { label: 'Raw', value: RepositoryFormatTypes.Raw }
+  { label: 'NuGet', value: RepositoryFormatTypes.NuGet }
+  // { label: 'Raw', value: RepositoryFormatTypes.Raw }
```
Comment (vikrant-harness) [inline src/modules/70-pipeline/utils/stageHelpers.ts:83]: Commented code?

Comment (satyam-harness) [reply src/modules/70-pipeline/utils/stageHelpers.ts:83]: yes, intentionally

**Review by vikrant-harness — APPROVED** [review]

**Pull request merged**